components/mgmtworker/scripts/configure.py

```diff
@@ -77,7 +77,7 @@ def prepare_snapshot_permissions():
     utils.sudo(['chmod', '-R', 'g+rw', utils.SSL_CERTS_TARGET_DIR])
     utils.sudo(['chmod', 'g+rw', dirname(utils.SSL_CERTS_TARGET_DIR)])
 
-    utils.sudo(['/opt/cloudify/snapshot_permissions_fixer'])
+    utils.sudo(['/opt/cloudify/mgmtworker/snapshot_permissions_fixer'])
```

Comment:
Please update this:
https://github.com/cloudify-cosmo/cloudify-manager/blob/ab855a6bef46c4de2ad8970d254ed2d1ff88d818/workflows/cloudify_system_workflows/snapshots/snapshot_restore.py#L98